tests/conftest.py

```diff
@@ -149,3 +149,11 @@ def create_bundle_archives(request, tmp_path: PosixPath) -> List[str]:
                 archive.add(license_fp, arcname=config[0]['license'])
         archives.append(str(archive_path))
     return archives
+
+
+@pytest.fixture(scope="session")
+def adcm_image_tags(cmd_opts) -> Tuple[str, str]:
+    """Get tag parts of --adcm-image argument (split by ":")"""
+    if not cmd_opts.adcm_image:
+        pytest.fail("CLI parameter adcm_image should be provided")
+    return tuple(cmd_opts.adcm_image.split(":", maxsplit=2))  # type: ignore
```

Comment:
see https://github.com/arenadata/adcm_pytest_plugin/pull/65

Reply:
Changed to `parse_repository_tag`